tile.yml

```diff
@@ -154,8 +140,28 @@ forms:
     label: PCF foundation name
     description: A user friendly PCF foundation name. This is included as a point tag on the metrics sent to Wavefront.
     optional: false
-  - name: pcf_skipSslValidation
+  - name: pcf_skip_ssl_validation
     type: boolean
     label: Skip SSL Verification
     default: false
     description: Useful for testing environments where SSL is not properly setup. Should not be used in prod environments.
+  - name: filter_metrics_white_list
+    type: string_list
+    label: Metrics White List
+    optional: true
+    description: Comma separated list of glob patterns. Only metrics with names matching the whitelist are reported.
+  - name: filter_metrics_black_list
+    type: string_list
+    label: Metrics Black List
+    optional: true
+    description: Comma separated list of glob patterns. Metrics with names matching the blacklist are dropped.
+  - name: filter_tags_white_list
```

Comment:
@laullon let's remove the tag based filtering options from the tile. We can leave it in the nozzle code. But won't be supported at the tile level.